enclave/core/tracee.c

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <openenclave/bits/safecrt.h>
+#include <openenclave/bits/safemath.h>
+#include <openenclave/bits/types.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/enclavelibc.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/sgxtypes.h>
+#include <openenclave/internal/trace.h>
+#include <openenclave/internal/utils.h>
+
+#define DEFAULT_LOGGING_FLAGS OE_LOG_FLAGS_NONE
+#define DEFAULT_LOGGING_LEVEL OE_LOG_LEVEL_NONE
+
+static oe_log_filter_t g_active_log_filter = {0};
```

Comment:
Can you address the naming of global variables. @anitagov recently did a pass

Reply:
Since this is static global, it is restricted to the file and you can use the \_oe_\ prefix if you wish. But this isn't mandatory. 
Refer to my issue #1093 which is removing the \_oe_\ prefix - You cannot define any identifiers in global scope whose names begin with an underscore, because they may conflict with hidden (private) library definitions

Reply:
Since this is static global, it is restricted to the file and you can use the _oe_ prefix if you wish. But this isn't mandatory.
Refer to my issue #1093 which is removing the _oe_ prefix - You cannot define any identifiers in global scope whose names begin with an underscore, because they may conflict with hidden (private) library definitions